cmake/configure/configure_2_trilinos.cmake

```diff
@@ -227,6 +262,10 @@ MACRO(FEATURE_TRILINOS_CONFIGURE_EXTERNAL)
   SET(DEAL_II_EXPAND_TRILINOS_MPI_VECTOR "TrilinosWrappers::MPI::Vector")
   IF (TRILINOS_WITH_MPI)
     SET(DEAL_II_EXPAND_EPETRA_VECTOR "LinearAlgebra::EpetraWrappers::Vector")
+    IF (${DEAL_II_TRILINOS_WITH_TPETRA})
+      SET(DEAL_II_EXPAND_TPETRA_VECTOR_DOUBLE "LinearAlgebra::TpetraWrappers::Vector<double>")
+      SET(DEAL_II_EXPAND_TPETRA_VECTOR_FLOAT "LinearAlgebra::TpetraWrappers::Vector<float>")
+    ENDIF()
```

Comment:
We don't need to expand for complex number?

Reply:
Now, we do. We might then check for support of `complex` values in `Tpetra` though.